README.md

```diff
@@ -1,35 +1,35 @@
 # Hello World
 ## 1.Try to solve "Helloworld" on codecheck
-You are in an Open Challenge as tutorial.
-You can learn how to use codecheck through taking this tutorial challenge.
+This is an Open Challenge as well as a tutorial.
+You can learn how to use codecheck by taking this tutorial.
 
-### 1-1.  Make sure "Deadline" and "Time Limit"
-After opening a challenge, you can see the detail of the challenge like this.  
-And you can see that the Deadline or Time limit is displayed on this page.  
-Please keep in mind that you can not submit your answer after the time is over.
+### 1-1.  Keep "Deadline" and "Time Limit" in mind.
+After openinig a challenge, it is possible to see the detail page of a challenge.
```

Comment:
After opening a challenge, you **will** see the detail page of **the** challenge.